### PR TITLE
Replace static imports where applicable

### LINKED ISSUE #202: Clean up imports
With the very large amount of constants in the codebase, using static imports makes the code very unmaintainable for a number of reasons, among which, is the fact that it is unclear which constants were declared locally and which are imported, e.g. those that are defined in `CommandTestUtil`.

Proposal: where appropriate, let us replace the static imports with qualified imports instead.

### PR DESCRIPTION
**Pull request by RichDom2185**
Closes #202. Please refer to issue for more details.

### CODE REVIEW

File: src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java [hunk 1]
```diff
@@ -4,9 +4,6 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.foodrem.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 import static seedu.foodrem.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
```
Comment (yixiann) [inline src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java:6]: Was wondering whether this is supposed to be replace with this

import static seedu.foodrem.commons.core.Messages;

**Review by yixiann — COMMENTED** [review]
LGTM

File: src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java (hunk 1 above)
Comment (RichDom2185) [reply src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java:6]: Both ways are possible. When it's clear and there's only a small amount of imports, sometimes we just use static imports so we don't have to call the constants by their fully qualified names. I chose to keep the static imports here.

**Review by yixiann — APPROVED** [review]

File: src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java (hunk 1 above)
Comment (yixiann) [reply src/test/java/seedu/foodrem/logic/parser/FoodRemParserTest.java:6]: I see. Thanks for the explanation!!

**Pull request merged**